ceph_deploy/hosts/debian/install.py

```diff
@@ -30,11 +30,11 @@ def install(distro, version_kind, version, adjust_repos, **kw):
         distro.packager.add_repo_gpg_key(gpg.url(key, protocol=protocol))
 
         if version_kind == 'stable':
-            url = 'http://download.ceph.com/debian-{version}/'.format(
+            url = 'https://download.ceph.com/debian-{version}/'.format(
```

Comment:
...and I guess we should do something to make sure that package is installed in general in the tool.